modules/UpgradeWizard/uw_utils.php

```diff
@@ -733,30 +744,27 @@ function upgradeUWFiles($file)
     }
     // upgradeWizard
     if (file_exists("$from_dir/modules/UpgradeWizard")) {
-        $allFiles[] = findAllFiles("$from_dir/modules/UpgradeWizard", $allFiles);
+        $allFiles[] = findAllFiles("$from_dir/modules/UpgradeWizard", array());
```

Comment:
Short array syntax is preferred:
```suggestion
        $allFiles[] = findAllFiles("$from_dir/modules/UpgradeWizard", []);
```